src/main/kotlin/no/nav/omsorgspenger/personopplysninger/pdl/PdlDomene.kt

```diff
@@ -34,4 +34,9 @@ data class PdlErrorExtension(
 fun hentPersonInfoQuery(fnr: Set<String>): PersonInfoGraphqlQuery {
     val query = GraphqlQuery::class.java.getResource("/pdl/hentPersonBolkInfo.graphql").readText().replace("[\n\r]", "")
     return PersonInfoGraphqlQuery(query, Variables(fnr.toList()))
+}
+
+fun hentFamilierelasjonQuery(fnr: String): PersonInfoGraphqlQuery {
+    val query = GraphqlQuery::class.java.getResource("/pdl/hentRelasjonerBolkInfo.graphql").readText().replace("[\n\r]", "")
```

Comment:
gjelder forsåvidt for begge funksjonene her, men kan du ikke flytte ut det å lese fra resources til et private objekt inne i fila her, så henter man det fra resources èn gang istedenfor for hver request.
Og om du markerer variabelen i objektet med `@JvmStatic` så skal det gjøres på oppstart